File: app/src/main/java/com/artear/multitrackerandroid/trackers/type/view/MyView.java
package com.artear.multitrackerandroid.trackers.type.view;

import com.artear.multitracker.contract.send.TrackerView;


public class MyView implements TrackerView {

    private String name;

    public MyView(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "< MyView - name = " + name + " >";
    }
}

File: app/src/main/java/com/artear/multitrackerandroid/trackers/type/view/MyView.kt
package com.artear.multitrackerandroid.trackers.type.view

import com.artear.multitracker.contract.send.TrackerView


class MyView(private val name: String) : TrackerView {

    override fun toString(): String {
        return "< MyView - name = $name >"
    }
}
